components/places/src/import/ios/history.rs

```diff
@@ -132,6 +136,15 @@ lazy_static::lazy_static! {
         WHERE url IS NOT NULL"
    ;
 
+    // Unfortunately UPDATE FROM is not available until sqlite 3.33
+   // however, iOS does not ship with 3.33 yet as of the time of writing.
+   static ref UPDATE_PLACES_TITLES: &'static str =
```

Comment:
I'm looking for more efficient ways to do this... ideally it would be in the same `INSERT OR IGNORE` below it, but that we want to ignore any issues there.

`INSERT OR REPLACE` would be cool, but we don't want to be deleting old entried and inserting new ones since lots of tables depend on the values of the places table..

This now isn't too bad, and is very far from being the bottleneck (I intentionally do it before the `FILL_MOZ_PLACES` because if we do it after, the joins will be painful and it will become the bottleneck)